src/streamlink/plugins/cinergroup.py

```diff
@@ -22,14 +23,42 @@
         |
         haberturk\.(?:com|tv)(?:/tv)?/canliyayin
         |
-        showmax\.com\.tr/canliyayin
+        bloomberght.com/tv
         |
-        showturk\.com\.tr/canli-yayin/showturk
+        showmax\.com\.tr/(canli-yayin|canliyayin)
+        |
+        showturk\.com\.tr/(canli-yayin|canliyayin)(/showtv)?
     )/?
 """, re.VERBOSE))
 class CinerGroup(Plugin):
-    def _get_streams(self):
-        stream_url = self.session.http.get(self.url, schema=validate.Schema(
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._page = None
+
+    @property
+    def page(self):
+        if self._page is None:
+            self._page = self.session.http.get(self.url, schema=validate.Schema(
+                validate.parse_html(),
+            ))
+        return self._page
+
+    def _get_live_url(self):
+        schema = validate.Schema(
+            validate.xml_xpath_string(".//script[contains(text(), 'videoUrl')]/text()"),
+            validate.none_or_all(
+                re.compile(r"""videoUrl\s*=\s*(?P<q>['"])(?P<url>.+?)(?P=q)"""),
+                validate.none_or_all(
+                    validate.get("url"),
+                    validate.url(),
+                ),
+            ),
+        )
+        live_url = schema.validate(self.page) 
+        return live_url
```

Comment:
Values that get returned should not be defined as a variable first. Patterns like this have just been cleaned up in the entire code base in c32952c17b23b0e797f6366b2aa64e8587b56e23